thefuck/rules/brew_cask_dependency.py

```diff
@@ -6,7 +6,8 @@
 @for_app('brew')
 def match(command):
     return (u'install' in command.script_parts
-            and u'brew cask install' in command.output)
+            and u'brew cask install' in command.output
+            and u'brew cask reinstall' in command.output)
```

Comment:
From what I understand, that should be an `or` rather than an `and`.

Reply:
An updated test would certainly help here!